scripts/localeNames.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   'am-ET': 'Amharic',
+  ar: 'Arabic',
   'ar-EG': 'Arabic (Egypt)',
   'ar-SA': 'Arabic (Saudi Arabia)',
   'ar-SD': 'Arabic (Sudan)',
```

Comment:
Would this work https://github.com/mui/material-ui/issues/31847#issuecomment-2536857376?

```suggestion
  ar: 'Standard Arabic',
```

Reply:
@0xenn bumping this PR and reminding of the proposed solutions to certain translations and this suggestion for the language name, which sounds logical to me. 👍

Reply:
@LukasTy i think it's sonds good ar: 'Standard Arabic'. Shall i make changes and raise pr.

Reply:
@0xenn I have applied this suggestion, it will fail the CI as `l10n` needs to be re-run now. 😉 
Could you check the existing suggestions in this PR, apply the ones that you agree with and we can go from there? 👍 
Here is a screenshot with the suggestion for the most optimal flow:
<img width="864" alt="Screenshot 2025-01-10 at 11 54 10" src="https://github.com/user-attachments/assets/be3551fd-8aba-4798-824d-6d0aa5a7ff98" />